todo_app/templates/index.html

```diff
@@ -1,17 +1,21 @@
 {% extends "layout.html" %}
 {% block title %}To-Do App{% endblock %}
-
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
-
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
+      <h5>ID - Status - Title</h5>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for itm in items %} 
+          <li>{{ itm.id }} - {{itm.status}} - {{itm.title}}</li> 
+        {% endfor %} 
+        <form action="#" method="POST">
+          <input type="Submit" value="AddItems">
```

Comment:
As above, this would be clearer as a link rather than a form that POSTs, e.g. using a href:
```python
<a href="/additem">Add Item</>
```
Or if you prefer the form appearance then just use it to get the relevant page straight off:
```python
        <form action="/additem" method="GET">
          <input type="Submit" value="AddItems">
        </form>
```